big_o/test/test_reports.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+import big_o
+from big_o import reports
+
+def find_max(x):
+    """Find the maximum element in a list of positive integers."""
+    max_ = 0
+    for el in x:
+        if el > max_:
+            max_ = el
+    return max_
+
+class TestReport(unittest.TestCase):
+
+    def test_report_return(self):
+        positive_int_generator = lambda n: big_o.datagen.integers(n, 0, 10000)
+        best, others = big_o.big_o(find_max, positive_int_generator, n_repeats=100, return_raw_data=True)
```

Comment:
It seems overkill to define the function `find_max` and execute it 100 times just to test that the report function returns a string. Please simplify the test, e.g. with `big_o.big_o(lambda n: 1, ...)` or something simple like that

Reply:
and remove the `find_max` function

Reply:
Done